### PR TITLE
Update Doc: Jenkins System Settings

### LINKED ISSUE #1483: [Have video] Add a tutorial about how to config Jenkins via CasC
See also https://github.com/jenkinsci/configuration-as-code-plugin

/area devops
/priority medium
/help

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

Fix #1483.

### CODE REVIEW

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

File: content/en/docs/devops-user-guide/how-to-use/jenkins-setting.md [hunk 1]
```diff
@@ -8,35 +8,43 @@ Weight: 11240
...
 
 This tutorial demonstrates how to set up Jenkins and reload configurations on the Jenkins dashboard.
 
 ## Prerequisites
 
 You have enabled [the KubeSphere DevOps System](../../../pluggable-components/devops/).
 
+## Jenkins Configuration as Code
+
+KubeSphere has Jenkins Configuration as Code plugin installed by default to allow you to define the desired state of your Jenkins configuration through YAML files and makes it easy to reproduce your Jenkins configurations including plugin configurations. You can find descriptions about specific Jenkins configurations and example YAML files [in this directory](https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos).
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/jenkins-setting.md:21]: has Jenkins Configuration as Code plugin installed
->
has the Jenkins Configuration as Code plugin installed

File: content/en/docs/devops-user-guide/how-to-use/jenkins-setting.md [hunk 2]
```diff
@@ -8,35 +8,43 @@ Weight: 11240
...
+Besides, you can find the `formula.yaml` file in the repository [ks-jenkins](https://github.com/kubesphere/ks-jenkins), where you can view plugin versions and customize these versions based on your needs.
+
+![plugin-version](/images/docs/devops-user-guide/using-devops/jenkins-system-settings/plugin-version.png)
+
 ## Modify the ConfigMap
 
-It is recommended that you configure Jenkins in KubeSphere through Configuration-as-Code (CasC). The built-in Jenkins CasC file is stored as a [ConfigMap](../../../project-user-guide/configuration/configmaps/).
+It is recommended that you configure Jenkins in KubeSphere through Configuration as Code (CasC). The built-in Jenkins CasC file is stored as a [ConfigMap](../../../project-user-guide/configuration/configmaps/).
 
 1. Log in to KubeSphere as `admin`. Click **Platform** in the top left corner and select **Cluster Management**.
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/jenkins-setting.md:31]: the top left corner
->
the top-left corner

File: content/en/docs/devops-user-guide/how-to-use/jenkins-setting.md [hunk 3]
```diff
@@ -8,35 +8,43 @@ Weight: 11240
...
 
-   ![cluster-management](/images/docs/devops-user-guide/using-devops/jenkins-system-settings/cluster-management.jpg)
+   ![cluster-management](/images/docs/devops-user-guide/using-devops/jenkins-system-settings/cluster-management.png)
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/jenkins-setting.md:33]: This image is not necessary. Remove it. Note that you only need to put images for key steps in a document.

**Comment by Sherlock113** [comment]
Please solve the conflict.

**Pull request merged**